tests/integration/test_reddit.py

```diff
@@ -37,6 +36,55 @@ def test_live_create(self, _):
             assert isinstance(live, LiveThread)
             assert live.title == 'PRAW Create Test'
 
+    def test_live_info__contain_invalid_id(self):
+        ids = ['3rgnbke2rai6hen7ciytwcxadi',
+               'LiveUpdateEvent_sw7bubeycai6hey4ciytwamw3a',  # invalid
+               't8jnufucss07']  # NBA
+        gen = self.reddit.live.info(ids)
+        with self.recorder.use_cassette('TestReddit.test_live_info'
+                                        '__contain_invalid_id'):
+            threads = list(gen)
+        assert len(threads) == 2
+
+    @mock.patch('time.sleep', return_value=None)
+    def test_live_info(self, _):
+        ids = """
+        ta40aifzobnv ta40l9u2ermf ta40ucdiq366 ta416hjgvbhy ta41ln5vsyaz
+        ta42cyzy1nze ta42i49806k0 ta436ojd653m ta43945wgmaa ta43znjvza9t
+        ta4418kxie3z ta44mk0nllhm ta45j0yvww9t ta4613lzdh8q ta46l8k86jt9
+        ta47qua0xu3n ta489fm9515p ta48ml5k1uk9 ta48zy4jzjcb ta49irvwndau
+        ta49upckgoyw ta4a02h9ynsb ta4aa4lrgvst ta4alauoi8ws ta4aqyacr70u
+
+        ta4ekdk6m5g2 ta4ezvoc49gy ta4f3iv06c1n ta4ffvliq5l7 ta4fib9lx3zx
+        ta4gka0ll41h ta4h89f6isfg ta4ht7s8he49 ta4i1eb564ar ta4imxhap4fg
+        ta4iu3g9whtk ta4j3o05j0d3 ta4kloqi6csg ta4m6kj44dql ta4mlqtihiil
+        ta4ng30l3fz1 ta4nldsjimhu ta4pd78tuk29 ta4prwyy1w9i ta4pvu8y6f8o
+        ta4ray2odqub ta4rua4oe6a1 ta4tk9fwjgz1 ta4trgqw6mmx ta4tv3sen7u4
+
+        ta4uyh0fnc0a ta4v54gnggcl ta4v5cm004z1 ta4vortaefna ta4wqym9d0v3
+        ta4wsuouxjtm ta4x7jr9v0fn ta4yast5e96b ta4z337yzlgu ta4zo9zzo9ui
+        ta507u2euo3w ta50exn0mtx1 ta51x2crezff ta52ch48gn6l ta53ijowvc6z
+        ta53iy196uod ta541cz1hfb4 ta54n0ncx8pc ta55ytfmre2g ta581bybyjwi
+        ta59gcidn2ym ta59mkwnrd43 ta5ar22wzi2w ta5awwo42ibb ta5dhmvylw0l
+
+        ta5hipd6wahr ta5itb7clg3s ta5nlm09y8kb ta5nm0f831x1 ta5oavbflorf
+        ta5rnv18s85o ta5ru6ysh254 ta5sfz02nc8b ta5syawj086b ta5t41osygln
+        ta5uy3ynoo4a ta5w0seb1xfy ta5wddbh0ln0 ta5zmjzuijwo ta617ozbmxhb
+        ta64q6pjz2bs ta696fdie4ne ta6bmog7gvoq ta6f9y7sdzru ta6j838d2wjn
+        ta6l4q5c17fd ta6ofypk3yp2 ta6sjmjt1aeb ta6sqhgyv41q ta70eezhz50r
+
+        ta72azs1l4u9 ta74r3dp2pt5 ta7pfcqdx9cl ta8zxbt2sk6z ta94nde51q4i
+        """.split()
+        gen = self.reddit.live.info(ids)
+        with self.recorder.use_cassette('TestReddit.test_live_info'):
+            threads = list(gen)
+        assert len(threads) > 100
+        assert all(isinstance(thread, LiveThread) for thread in threads)
+        # output may not reflect input order
+        thread_ids = [thread.id for thread in threads]
+        assert thread_ids != ids
+        assert sorted(thread_ids) == ids
```

Comment:
I suspect [this validator](https://github.com/reddit/reddit-plugin-liveupdate/blob/bf9c0e692a5bf3f907a986f56443c5604df6e8a7/reddit_liveupdate/validators.py#L49) gets data from DB as a dict (not list) because `return_dict=False` is not specified, but it's pure guess.